src/main/scala/beam/utils/analysis/PathTraversalAnalysis.scala

```diff
@@ -0,0 +1,52 @@
+package beam.utils.analysis
+
+import java.io.Closeable
+
+import beam.agentsim.events.PathTraversalEvent
+import beam.utils.EventReader
+import beam.utils.csv.CsvWriter
+import org.matsim.api.core.v01.events.Event
+import org.matsim.core.network.NetworkUtils
+import org.matsim.core.network.io.MatsimNetworkReader
+
+object PathTraversalAnalysis {
```

Comment:
rename: ByStopTransitOccupancyCSV

Reply:
can we integrate this in beam, e.g. create in iterations folder each iteration x.transitOccupancyByStop.csv

Reply:
rename method to TransitOccupancyByStopAnalysis